addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -62,12 +62,17 @@
                     <t t-set="show_name" t-value="True"/>
                     <t t-set="compact" t-value="True"/>
                 </t>
-                <small class="text-muted">
-                    <i class="fa fa-calendar me-1"/><span t-field="question.write_date" t-options='{"format":"short"}'/>
+                <small class="text-muted" title="Last Activity">
+                    <i class="fa fa-calendar me-1"/>
+                    <span t-if="not answer" t-field="question.last_activity_date" t-options='{"format":"short"}'/>
+                    <span t-if="answer" t-field="question.parent_id.last_activity_date" t-options='{"format":"short"}'/>
```

Comment:
```suggestion
                    <span t-field="question.last_activity_date" t-options='{"format":"short"}'/>
```

Reply:
but whenever users edit their answers, a date will remain the same as an old date. Is it okay?